src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -4150,121 +2892,7 @@ void CalcISO15099WindowIntConvCoeff(EnergyPlusData &state,
         state.dataHeatBalSurf->SurfHConvInt(SurfNum) = state.dataHeatBal->LowHConvLimit;
 }
 
-RoofGeoCharacteristicsStruct getRoofGeometryInformation(EnergyPlusData &state)
```

Comment:
Moved this to `SurfaceGeometry` module. Why is roof geometry summary so detailed but facade geometry summary is not?